src/olympia/lib/es/management/commands/reindex.py

```diff
@@ -178,12 +179,17 @@ def add_alias_action(action, index, alias):
         for alias, module in modules.items():
             old_index = None
 
-            olds = ES.indices.get_aliases(alias, ignore=404)
-            for old_index in olds:
-                # Mark the index to be removed later.
-                to_remove.append(old_index)
-                # Mark the alias to be removed from that index.
-                add_alias_action('remove', old_index, alias)
+            try:
+                olds = ES.indices.get_aliases(alias)
+                for old_index in olds:
+                    # Mark the index to be removed later.
+                    to_remove.append(old_index)
+                    # Mark the alias to be removed from that index.
+                    add_alias_action('remove', old_index, alias)
+            except NotFoundError:
+                # If the alias dit not exist, ignore it, don't try to remove
```

Comment:
We should at least log the 404 imho to make sure we're able to see why things are happening or why they aren't.